rpc/test_utils.go

```diff
@@ -129,14 +147,15 @@ func createTestProtocol(i TestInterface) Protocol {
 		Methods: map[string]ServeHandlerDescription{
 			"add": {
 				MakeArg: func() interface{} {
-					return new(AddArgs)
+					ret := make([]AddArgs, 1)
```

Comment:
change this so that it's more like what we currently generate via the RPC generator